src/agoradatatools/etl/load.py

```diff
@@ -160,3 +160,21 @@ def dict_to_json(df: dict, staging_path: str, filename: str) -> str:
     json.dump(df_as_dict, temp_json, cls=NumpyEncoder, indent=2)
     temp_json.close()
     return temp_json.name
+
+
+def list_to_json(df: list, staging_path: str, filename: str) -> str:
```

Comment:
Nitpick: can we call the variable something other than `df` since it's a list? Though I guess the `dict_to_json` function does the same thing... hmm.